production/models.py

```diff
@@ -58,7 +58,8 @@ class Meta:
     description_en = models.TextField(blank=True, null=True)
 
     def __unicode__(self):
```

Comment:
Think of a fallback in case the title is empty (that would make an unclickable label such as " - ").